htslib/vcf.h

```diff
@@ -1411,7 +1411,7 @@ static inline int64_t bcf_dec_int1(const uint8_t *p, int type, uint8_t **q)
         *q = (uint8_t*)p + 4;
         return le_to_i32(p);
     } else if (type == BCF_BT_INT64) {
-        *q = (uint8_t*)p + 4;
+        *q = (uint8_t*)p + 8;
```

Comment:
This is an additional bug to the one I found.   I wonder why it wasn't breaking things.  Is this function only used when decoding the BCF file itself, in which case it's not triggerable as it's not actually a valid thing to have in the file?

(But still worth fixing as clearly the code is incorrect.)

Reply:
It was breaking things when reading BCF_BT_INT64 values from an unofficial 64-bit flavor of BCF.